addons/website/static/src/snippets/s_facebook_page/000.js

```diff
@@ -69,27 +51,13 @@ const FacebookPageWidget = publicWidget.Widget.extend(ObservingCookieWidgetMixin
             this.iframeEl.setAttribute("style", "border: none; overflow: hidden;");
             this.iframeEl.setAttribute("aria-label", _t("Facebook"));
             this.el.replaceChildren(this.iframeEl);
-            this._manageIframeSrc(this.el, src);
+            this.services.website_cookies.manageIframeSrc(this.el, src);
```

Comment:
FYI, I modified the params of the function. It directly takes `iframeEl` instead of a parent.